scripts/reformat_data.py

```diff
@@ -81,6 +81,8 @@ def main():
     # remove 'not-availables'
     df = df.replace('Not Available', np.nan)
 
+    df.to_csv(FILENAME_REFORMATED, index=0)
```

Comment:
http://www.emoji-cheat-sheet.com/
`:+1:`

On Thu, Mar 17, 2016 at 9:18 AM, Elijah Bernstein-Cooper <
notifications@github.com> wrote:

> In scripts/reformat_data.py
> <https://github.com/arich/hospital_readmission_server/pull/6#discussion_r56510404>
> :
>
> > @@ -81,6 +81,8 @@ def main():
> >      # remove 'not-availables'
> >      df = df.replace('Not Available', np.nan)
> >
> > +    df.to_csv(FILENAME_REFORMATED, index=0)
>
> How did you do the thumbs up?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/arich/hospital_readmission_server/pull/6/files/af1b6dd96ee4f4b2815283e1be5e197a1a272f02#r56510404>
>